tests/stl_import_data/conf.out/pkexcept

```diff
@@ -0,0 +1 @@
+None
```

Comment:
Is this file necessary? I deleted the entire conf.out directory without apparent effect